bpf/kprobe_pwru.c

```diff
@@ -393,6 +391,17 @@ PWRU_ADD_KPROBE(3)
 PWRU_ADD_KPROBE(4)
 PWRU_ADD_KPROBE(5)
 
+SEC("kprobe/skb_by_stackid")
+int kprobe_skb_by_stackid(struct pt_regs *ctx) {
+	u64 stackid = get_stackid(ctx);
+
+	struct sk_buff **skb = bpf_map_lookup_elem(&stackid_skb, &stackid);
+	if (skb && *skb)
```

Comment:
Why `if (skb)` is not enough?

Reply:
You are right, it's MS Copilot! Let me fix it.

Reply:
Okay.. you merged ...